tests/integration/modifiers/did-intersect-test.js

```diff
@@ -344,4 +352,46 @@ module('Integration | Modifier | did-intersect', function (hooks) {
 
     assert.equal(this.newExitStub.callCount, 1, 'new exit callback is called');
   });
+
+  module('modifier accepts `isObserving` argument', function () {
+    test('with a truth(y) value', async function (assert) {
+      assert.expect(2);
```

Comment:
Prefer not to use `assert.expect` for tests where the assertions are always called synchronously rather than in callbacks.

Reply:
We can fix this in another PR, since it needs to be updated everywhere.

Reply:
Filed #398

Reply:
Heh, I am 100% in agreement; I added it solely to match the existing tests I saw. 😅